src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -10,6 +10,15 @@
      * Результат: 21</p>
      */
     public int getFibonacciNumber(int index) {
-        return 0;
+        if (index == 0) {
+            return 0;
+        }
+        if (index == 1 || index == -1) {
+            return 1;
+        }
+        if (index < 0) {
+            return getFibonacciNumber(index + 2) + getFibonacciNumber(index + 1);
+        }
```

Comment:
why do we need to check for negative indices?

Reply:
done